src/packages/cli/src/Version.ts

```diff
@@ -82,6 +82,10 @@ export class Version implements Command {
       ['Migration Engine', this.printBinaryInfo(migrationEngine)],
       ['Introspection Engine', this.printBinaryInfo(introspectionEngine)],
       ['Format Binary', this.printBinaryInfo(fmtBinary)],
+      [
+        'Default Engines Hash',
```

Comment:
```suggestion
        'Engines Commit',
```
Would be more accurate?

Reply:
The idea behind "Default" was that you can overwrite it by using ENV vars. So this would show the version, that this CLI version would use by default.

Hash/Commit is pretty identical 👍

Reply:
So in the end this is more accurate I guess:
```suggestion
        '@prisma/engines',
```

Reply:
Or we keep `Default` like you suggested